StratisCore.UI/src/app/shared/services/interfaces/signalr-events.i.ts

```diff
@@ -0,0 +1,23 @@
+import { GeneralInfo, StakingInfo } from "@shared/services/interfaces/api.i";
```

Comment:
Not a big one, but we have a mixture of " and ', in most old components it is ' and TSLint prefers '